src/core/dev_api/openvino/op/group_query_attention.hpp

```diff
@@ -11,6 +11,7 @@ namespace ov::op::internal {
 class OPENVINO_API GroupQueryAttention : public Op {
 public:
     OPENVINO_OP("GroupQueryAttention");
+    ~GroupQueryAttention() override;
```

Comment:
that means we should always remember do not forget to add it.
Can we do this in a bit robust way, for example, using macro `OPENVINO_OP` will add it automatically?

Reply:
Good hint .. @mitruska imo it should be addressed in a separate PR (or set of PRs).

Reply:
I have the same concern, when adding new op, there is no validation to check if the destructor has been added.
So it would be hard to maintain, macro sounds like a good idea, but as the proposed solution requires hpp and cpp update, extension of OPENVINO_OP likely will not handle cpp changes.

The PR changes itself look functionally correct, but if "more robust way" would be applied as next step, then a lot of reverting will be needed again.